### PR TITLE
VFE - Deserters patch

### LINKED ISSUE #2738: [Patch Request]:Vanilla Factions Expanded - Deserters
**Mod name:**
Vanilla Factions Expanded - Deserters
**Link to mod download (preferably Steam workshop or Github):**
https://github.com/Vanilla-Expanded/VanillaFactionsExpanded-Deserters
**Description:**
One of the features of this mod is the ability to purchase charge weapons via the console. It would be great to be able to purchase charge ammo as well.

**Content:**
What does the new mod contain? Check all that apply.
- [ ] New weapons.
- [ ] New animals, humanoids, or other creatures.
- [x] New factions.
- [x] New armor or apparel.

**Complete the following checklist before submitting a request.**
I hereby verify that I have done all of the following:
- [x] Searched the Steam workshop, Discord, and Github for an existing patch and not found one.
- [x] Confirmed there are no other open Patch Requests for this mod.
- [x] Confirmed the requested mod is not known to be fundamentally incompatible with Combat Extended.
- [x] **Have completed this template accurately and completely.**
- [x] **Understand that putting in a Patch Request does not guarantee a patch will be made.**
- [x] **Agree that I will not directly contact contributors to ask about patch requests.**

**Putting in request does not guarantee a patch will be made. Do not directly contact team members about Patch Requests. For smaller mods, consider reviewing the guide and making a patch yourself. If you wish to create a patch for any requested mod, there is no need to ask for permission.**

### PR DESCRIPTION
**Pull request by SamaelGray**
## Changes

- What it says on the tin.
Closes #2738 

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (specify how long)

### CODE REVIEW

**Review by Thorbane — APPROVED** [review]

**Comment by HeraldOfTzeentch** [comment]
appears the xml for the turrets throws errors upon startup and isnt working

**Review by albertbokor — APPROVED** [review]

**Review by mlg6662 — COMMENTED** [review]
hi :3

**Review by N7Huntsman — APPROVED** [review]

**Review by Safairette — CHANGES_REQUESTED** [review]
One oversight to be fixed and one potential balancing issue.
Otherwise looks good to me.

File: Patches/Vanilla Factions Expanded - Deserters/RangedSpacer.xml [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+								<armorPenetrationBlunt>18</armorPenetrationBlunt>
+								<spreadMult>8.9</spreadMult>
+							</projectile>
+						</ThingDef>
+
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VFED_Gun_ChargeCycler"]/verbs</xpath>
```
[the remark this answers is not in the document]
Comment (Safairette) [reply Patches/Vanilla Factions Expanded - Deserters/RangedSpacer.xml]: Removed the duplicate patch

**Review by N7Huntsman — APPROVED** [review]

**Review by N7Huntsman — APPROVED** [review]

File: Patches/Vanilla Factions Expanded - Deserters/Contraband.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+					<category>VFED_Imperial</category>
+						<intelCost>5</intelCost>
+					<useCriticalIntel>false</useCriticalIntel>
+					<countMult>100</countMult>
+				</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>Defs/ThingDef[defName="Ammo_5x35mmCharged"]</xpath>
```
Comment (SamaelGray) [inline Patches/Vanilla Factions Expanded - Deserters/Contraband.xml:50]: Ammo for a weapon that is unusable in CE?

**Review by Safairette — APPROVED** [review]
Contraband doesn't respect generic mode at the moment, but otherwise seems to work fine now.

**Review by N7Huntsman — APPROVED** [review]

**Comment by SamaelGray** [comment]
The CEG charge sniper and its ammo are not added to the contraband list yet.

**Comment by N7Huntsman** [comment]
Whoops, was working on spotty wifi and the commit got hung up. There we go.

**Review by Safairette — APPROVED** [review]

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**